app/src/main/java/com/kamiz/rappiapp/ui/login/LoginFragment.kt

```diff
@@ -4,16 +4,45 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
 import com.kamiz.rappiapp.R
+import com.kamiz.rappiapp.databinding.FragmentLoginBinding
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class LoginFragment : Fragment() {
```

Comment:
Please create a BaseFragment with the following functions:
setupObserver()
setupViews()